app/components/pages/createPoll/index.js

```diff
@@ -0,0 +1,234 @@
+import { Toast } from '../../../utils/helper';
+
+const CreatePollPage = ({
+  heading,
+  inputLabelName,
+  inputLabelQuestion,
+  buttonName,
+  optionButtonName,
+}) => {
+  const classes = useStyles();
+  const upperLimitForOptions = 4;
+  return (
+    <WrapInBreadcrumbs>
+      <WrapInCard>
+        <Formik
+          initialValues={{
+            options: [''],
+            textFieldName: '',
+            textFieldQuestion: '',
+            startDate: undefined,
+            endDate: undefined,
+          }}
+          onSubmit={() => {
+            Toast({
+              icon: 'success',
+              title: `Poll created successfully`,
+            });
+          }}
+          validationSchema={Yup.object().shape({
+            textFieldName: Yup.string().required('Required'),
+            textFieldQuestion: Yup.string().required('Required'),
+            startDate: Yup.date()
+              .required('Required')
+              .default(() => new Date()),
+            endDate: Yup.date()
+              .required('Required')
+              .when(
+                'startDate',
+                (startDate, schema) =>
+                  startDate &&
+                  schema.min(startDate, 'Must be greater than start date')
+              ),
+          })}
+          render={({ values }) => (
+            <Form>
+              <Box display="flex" justifyContent="center" alignItems="center">
+                <Box pt={15} px={2} width={[1, 1 / 2]}>
+                  <Box
+                    flex="1"
+                    display="flex"
+                    justifyContent={['center']}
+                    fullWidth
+                  >
+                    <Typography>
+                      <H5>{heading}</H5>
+                    </Typography>
+                  </Box>
+
+                  <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    flexDirection={['column', 'row']}
+                  >
+                    <Box width={[1, 1 / 2]} mt={10} px={3}>
+                      <Input
+                        OutlinedInputPlaceholder={inputLabelName}
+                        name="textFieldName"
+                        variant="outlined"
+                        Icon={PersonIcon}
+                        appendIcon
+                      />
+                    </Box>
+                    <Box flex="1" mt={10} px={3}>
+                      <CustomDatePicker
+                        name="startDate"
+                        label="Start Date"
+                        className={classes.CustomFullWidth}
+                      />
+                    </Box>
+                  </Box>
+
+                  <Box
+                    display="flex"
+                    justifyContent="space-between"
+                    flexDirection={['column', 'row']}
+                  >
+                    <Box flex="1" mt={10} px={3}>
+                      <CustomDatePicker
```

Comment:
use KeyboardDateTimePicker

Reply:
"CustomDatePicker" a reuseable commponent of keyboardDatePicker